### PR TITLE
Code style update and general clean up

### PR DESCRIPTION
**Pull request by sanduhrs**

### CODE REVIEW

File: src/GeocoderJsonConsumer.php [hunk 1]
```diff
@@ -72,7 +72,6 @@ public function getAddress($text) {
         foreach ($data->results as $result) {
           if (!empty($result->formatted_address)) {
             $formatted_address = $result->formatted_address;
-            // Names containing commas or quotes must be wrapped in quotes.
```
Comment (sanduhrs) [inline src/GeocoderJsonConsumer.php:70]: Why is that?

File: src/GeocoderJsonConsumer.php [hunk 2]
```diff
@@ -72,7 +72,6 @@ public function getAddress($text) {
...
             $matches[] = array(
               'value' => Html::escape($formatted_address),
               'label' => Html::escape($formatted_address),
```
Comment (sanduhrs) [inline src/GeocoderJsonConsumer.php:77]: IMHO Escaping the data from remote data sources is a must.

File: src/Controller/GeocoderController.php [hunk 3]
```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\geocoder_autocomplete\Controller\GeocoderController.
- */
```
Comment (itamair) [inline src/Controller/GeocoderController.php:5]: phpcs (in my phpstorm) says that is missing file doc comment, if we remove these comments lines.
Why remove?

File: src/GeocoderJsonConsumer.php [hunk 4]
```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains /Drupal/geocoder_autocomplete/GeocoderJsonConsumer.
- */
```
Comment (itamair) [inline src/GeocoderJsonConsumer.php:5]: phpcs (in my phpstorm) says that is missing file doc comment, if we remove these comments lines.
Why remove?

File: src/Plugin/Field/FieldWidget/GeocoderAutocomplete.php [hunk 5]
```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\geocoder_autocomplete\Plugin\Field\FieldWidget\GeocoderAutocomplete.
- */
```
Comment (itamair) [inline src/Plugin/Field/FieldWidget/GeocoderAutocomplete.php:5]: phpcs (in my phpstorm) says that is missing file doc comment, if we remove these comments lines.
Why remove?

File: src/GeocoderJsonConsumer.php [hunk 6]
```diff
@@ -1,13 +1,10 @@
...
 
 namespace Drupal\geocoder_autocomplete;
 
-use GuzzleHttp\ClientInterface;
 use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Component\Utility\Html;
```
Comment (itamair) [inline src/GeocoderJsonConsumer.php:6]: this class is never used in the file ...

File: src/Controller/GeocoderController.php (hunk 3 above)
Comment (sanduhrs) [inline src/Controller/GeocoderController.php:5]: Do you use the latest version of phpcs drupal standard?
I use Verison 8.x-2.7 from https://www.drupal.org/project/coder

File: src/GeocoderJsonConsumer.php (hunk 4 above)
Comment (sanduhrs) [inline src/GeocoderJsonConsumer.php:5]: Do you use the latest version of phpcs drupal standard?
I use Verison 8.x-2.7 from https://www.drupal.org/project/coder

File: src/Plugin/Field/FieldWidget/GeocoderAutocomplete.php (hunk 5 above)
Comment (sanduhrs) [inline src/Plugin/Field/FieldWidget/GeocoderAutocomplete.php:5]: Do you use the latest version of phpcs drupal standard?
I use Verison 8.x-2.7 from https://www.drupal.org/project/coder

File: src/GeocoderJsonConsumer.php (hunk 6 above)
Comment (sanduhrs) [inline src/GeocoderJsonConsumer.php:6]: It is used in line 77 and 78 of the file:
https://github.com/itamair/geocoder_autocomplete/pull/1/commits/ae5a4d0247ad8bdb0138db9ab75337714c68d57d

Comment (itamair) [inline src/GeocoderJsonConsumer.php:6]: Right.  Sorry ...

File: src/Controller/GeocoderController.php (hunk 3 above)
Comment (itamair) [inline src/Controller/GeocoderController.php:5]: tnx. Let me check ... once done, I will merge your pull request. Tnx so much for all this ...

Comment (itamair) [inline src/Controller/GeocoderController.php:5]: Ok. Didi it. You are right. last phpcs changed specifications on namespaced classes and file doc comments. Going to merge ...

**Pull request merged**